transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/Constraint.groovy

```diff
@@ -635,6 +635,28 @@ class SubSelectionConstraint extends Constraint {
     }
 }
 
+@Canonical
+class RelationConstraint extends Constraint {
```

Comment:
@forus The rest of the API uses underscores, here it is camel case, we probably want to change this class to use underscores.